FWCore/Framework/test/BuildFile.xml

```diff
@@ -36,6 +36,12 @@
   <use name="FWCore/Utilities"/>
 </bin>
 
+<test name="TestFWCoreFrameworkOptions0" command="run_testOptions.sh 0"/>
+<test name="TestFWCoreFrameworkOptions1" command="run_testOptions.sh 1"/>
+<test name="TestFWCoreFrameworkOptions2" command="run_testOptions.sh 2"/>
+<test name="TestFWCoreFrameworkOptions3" command="run_testOptions.sh 3"/>
+<test name="TestFWCoreFrameworkOptions4" command="run_testOptions.sh 4"/>
```

Comment:
@wddgit , these are good candidates of recently added feature of scram V3 (which I will present in one of core sw meetings). You can use either
```
<test name="TestFWCoreFrameworkOptions" command="run_testOptions.sh ${value}" for="0,4"/>
```
or
```
<test name="TestFWCoreFrameworkOptions" command="run_testOptions.sh ${value}" foreach="0,1,2,3,4"/>
```

and scram will define `TestFWCoreFrameworkOptions_N` tests

Reply:
I just pushed a commit using the new for loop in the BuildFile. It seems to work well. Thanks.